test/md_zip_view_test.cpp

```diff
@@ -566,47 +567,48 @@ TEST(MDZipView, derived_standard_deviation) {
 TEST(MDZipView, create_from_labels) {
   Dataset d;
   d.insert(Data::Value, "", Dimensions(Dim::X, 2), 2);
-  d.insert(Data::Int, "", {}, 1);
+  d.insert(Data::Variance, "", {}, 1);
   auto var = d.get(Data::Value);
   var[0] = 0.2;
   var[1] = 3.2;
 
-  ASSERT_ANY_THROW(zipMD(d, MDWrite(Data::Value), MDWrite(Data::Int)));
-  ASSERT_NO_THROW(zipMD(d, MDWrite(Data::Value), MDRead(Data::Int)));
-  auto view = zipMD(d, MDWrite(Data::Value), MDRead(Data::Int));
+  ASSERT_ANY_THROW(zipMD(d, MDWrite(Data::Value), MDWrite(Data::Variance)));
+  ASSERT_NO_THROW(zipMD(d, MDWrite(Data::Value), MDRead(Data::Variance)));
+  auto view = zipMD(d, MDWrite(Data::Value), MDRead(Data::Variance));
   auto it = view.begin();
   ASSERT_EQ(it->get(Data::Value), 0.2);
-  ASSERT_EQ(it->get(Data::Int), 0);
+  ASSERT_EQ(it->get(Data::Variance), 0);
   it++;
   ASSERT_EQ(it->get(Data::Value), 3.2);
-  ASSERT_EQ(it->get(Data::Int), 0);
+  ASSERT_EQ(it->get(Data::Variance), 0);
 }
 
 TEST(MDZipView, create_from_labels_with_name) {
   Dataset d;
   d.insert(Data::Value, "name", Dimensions(Dim::X, 2), 2);
-  d.insert(Data::Int, "name", {}, 1);
+  d.insert(Data::Variance, "name", {}, 1);
```

Comment:
We already had the `Variance` tag - i must be misunderstanding something. Why did these tests use `Int` as the tag?

Reply:
The actual tag in this test is irrelevant, as long as it is two different ones.